project/AkkaBuild.scala

```diff
@@ -1547,6 +1552,9 @@ object Dependencies {
     // For akka-http spray-json support
     val sprayJson   = "io.spray"                     %% "spray-json"                   % "1.3.1"       // ApacheV2
 
+    //For akka-http play-json support
+    val playJson    = "com.typesafe.play"            %% "play-json"                    % "2.3.8"       // ApacheV2
```

Comment:
@jroper do you have an opinion about what `play-json` version should be supported here (it's for a simple "support" trait for akka-http). Play `2.4` is in RC1 mode AFAICS, and akka-http is experimental, also in RC mode - would it make sense to depdend on `2.4-RC1` here instead then?

// cc @rkuhn